paper-input-behavior.html

```diff
@@ -170,6 +170,13 @@
       // HTMLInputElement attributes for binding if needed
 
       /**
+       * Bind this to the `<input is="iron-input">`'s `title` property.
+       */
```

Comment:
I don't think `title` needs to be defined as a property, since it's a native attribute (and therefore always present)